english/recipes/chicken/chicken-breasts.md

```diff
@@ -0,0 +1,25 @@
+Ingredients
+
+4 skinless, boneless chicken breast halves
+2 tablespoons olive oil
+1 tablespoon coarse sea salt
+1 pinch Creole seasoning, or to taste
+1 tablespoon water, or as needed
+
+
+Directions
+
+1.  Preheat convection oven to 400 degrees F (200 degrees C).
+
+2.  Rub chicken breasts with olive oil and sprinkle both sides with salt and Creole seasoning. Place chicken in a broiler pan.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.